cmd/bbolt/page_command.go

```diff
@@ -1,105 +1,97 @@
 package main
 
 import (
-	"bytes"
-	"flag"
 	"fmt"
-	"io"
 	"os"
-	"strings"
 
+	"github.com/spf13/cobra"
 	"go.etcd.io/bbolt/internal/common"
 	"go.etcd.io/bbolt/internal/guts_cli"
 )
 
-// pageCommand represents the "page" command execution.
-type pageCommand struct {
-	baseCommand
-}
+// NewPageCommand creates a new Cobra command for the "page" subcommand.
+func NewPageCommand() *cobra.Command {
+	var all bool
+	var formatValue string
```

Comment:
you can have those as global vars and use `newPageCommand()` as you did before :)